### ISSUE TITLE
Fatal error in launcher

### ISSUE THREAD

**Issue by dill-shower** [devilismyfriend/StableTuner#50]
I loaded the model into StableTuner, installed the concept, specified all the settings. I'm trying to run the training and I'm getting an error

loading text encoder: <All keys matched successfully>
copy scheduler/tokenizer config from: runwayml/stable-diffusion-v1-5
Diffusers model saved.

(ST) C:\StableTuner>accelerate "launch" "--mixed_precision=fp16" "scripts/trainer.py" "--model_variant=base"  "--disable_cudnn_benchmark"  "--use_text_files_as_captions"  "--sample_step_interval=500"  "--pretrained_model_name_or_path=C:\StableTuner\models\wd-1-3-penultimate-ucg-cont"  "--pretrained_vae_name_or_path="  "--output_dir=models/new_model"  "--seed=3434554"  "--resolution=512"  "--train_batch_size=24"  "--num_train_epochs=100"  "--mixed_precision=fp16"  "--use_bucketing"  "--aspect_mode=dynamic"  "--aspect_mode_action_preference=add"  "--use_8bit_adam"  "--gradient_checkpointing"  "--gradient_accumulation_steps=1"  "--learning_rate=3e-6"  "--lr_warmup_steps=0"  "--lr_scheduler=constant"  "--train_text_encoder"  "--concepts_list=stabletune_concept_list.json"  "--num_class_images=200"  "--save_every_n_epoch=5"  "--n_save_sample=1"  "--sample_height=512"  "--sample_width=512"  "--dataset_repeats=1"  "--sample_on_training_start"
Fatal error in launcher: Unable to create process using '"C:\ProgramData\miniconda3\python.exe"  "C:\Users\adam\AppData\Roaming\Python\Python310\Scripts\accelerate.exe" "launch" "--mixed_precision=fp16" "scripts/trainer.py" "--model_variant=base"  "--disable_cudnn_benchmark"  "--use_text_files_as_captions"  "--sample_step_interval=500"  "--pretrained_model_name_or_path=C:\StableTuner\models\wd-1-3-penultimate-ucg-cont"  "--pretrained_vae_name_or_path="  "--output_dir=models/new_model"  "--seed=3434554"  "--resolution=512"  "--train_batch_size=24"  "--num_train_epochs=100"  "--mixed_precision=fp16"  "--use_bucketing"  "--aspect_mode=dynamic"  "--aspect_mode_action_preference=add"  "--use_8bit_adam"  "--gradient_checkpointing"  "--gradient_accumulation_steps=1"  "--learning_rate=3e-6"  "--lr_warmup_steps=0"  "--lr_scheduler=constant"  "--train_text_encoder"  "--concepts_list=stabletune_concept_list.json"  "--num_class_images=200"  "--save_every_n_epoch=5"  "--n_save_sample=1"  "--sample_height=512"  "--sample_width=512"  "--
Press any key to continue... 

Os: windows 10
Tried reinstalling several times, removed conda, python and all env. Problem still exists
Model: waifu-diffusion. This problem with all models (try sd1.4, sd1.5 and SD2.1)

**Comment by devilismyfriend** [comment]
Did you run the installer bat

**Comment by dill-shower** [comment]
> Did you run the installer bat

Yes. And it was success.

[ ] xformers version 0.0.14.dev0 installed.
[+] bitsandbytes version 0.35.0 installed.
[ ] diffusers version 0.10.2 installed.
[+] transformers version 4.25.1 installed.
[+] torch version 1.12.1+cu116 installed.
[+] torchvision version 0.13.1+cu116 installed.
Checking StableTuner requirements...
Checking/upgrading existing torch/torchvision installation
[ ] xformers version 0.0.14.dev0 installed.
[+] bitsandbytes version 0.35.0 installed.
[ ] diffusers version 0.10.2 installed.
[+] transformers version 4.25.1 installed.
[+] torch version 1.12.1+cu116 installed.
[+] torchvision version 0.13.1+cu116 installed.
Can't find CUDNN in resources, trying main folder...
Checking for B&B files in C:\ProgramData\Anaconda3\envs\ST\Lib\site-packages\bitsandbytes
Copied B&B files to destination
Checking for CUDNN files in C:\ProgramData\Anaconda3\envs\ST\Lib\site-packages\torch\lib
Installing diffusers 0ca1724 commit
Updating requirements
Press any key to continue...

**Comment by dill-shower** [comment]
It appears to be a problem with accelerate. Went into conda env and tried to run

(ST) PS C:\StableTuner> accelerate

Fatal error in launcher: Unable to create process using '"C:\ProgramData\miniconda3\python.exe" "C:\Users\user\AppData\Roaming\Python\Python310\Scripts\accelerate.exe" ': ?? ??????? ????? ????????? ????.

**Comment by dill-shower** [comment]
Iw was problem with my PATH. Sorry
